libs/etterlatte-ktor/src/main/kotlin/RestModule.kt

```diff
@@ -77,14 +87,35 @@ fun Application.restModule(
         tokenValidationSupport(config = config)
     }
 
+    if (withMetrics) {
+        val collectorRegistry = CollectorRegistry.defaultRegistry
+
+        val registrySaks = PrometheusMeterRegistry(
+            PrometheusConfig.DEFAULT,
+            collectorRegistry,
+            Clock.SYSTEM
+        )
+        install(MicrometerMetrics) {
+            registry = registrySaks
+            meterBinders = listOf(
+                LogbackMetrics(),
+                JvmMemoryMetrics(),
```

Comment:
Usikker hvilken GC vi bruker - men - er det noen behov for JvmGcMetrics?

Reply:
Vet ikke, la det på igjen her https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/1710/commits/2a4e0a53618c89d06ba5c6e5e48c5bb901699762